src/components/NewAnalysisModal.js

```diff
@@ -0,0 +1,102 @@
+      isAnalysisMode: true,
+      workspace,
+      tool,
+      runtimes,
+      persistentDisks,
+      onDismiss,
+      onSuccess: _.flow(
+        withErrorReporting('Error creating runtime'),
+        Utils.withBusyState(setBusy)
+      )(async () => {
+        onSuccess()
+        await refreshRuntimes(true)
+      })
+    })
+
+    const renderNewGalaxyModal = () => h(NewGalaxyModalBase, {
+      isOpen: true,
+      isAnalysisMode: true,
+      workspace,
+      apps,
+      galaxyDataDisks,
+      onDismiss,
+      onSuccess: _.flow(
+        withErrorReporting('Error creating app'),
+        Utils.withBusyState(setBusy)
+      )(async () => {
+        onSuccess()
+        await refreshApps(true)
+      })
+    })
+
+    const NEW_JUPYTER_MODE = tools.Jupyter.label
+    const NEW_RSTUDIO_MODE = tools.RStudio.label
+    const NEW_GALAXY_MODE = 'Galaxy'
+    const getView = () => Utils.switchCase(viewMode,
+      [NEW_JUPYTER_MODE, () => renderNewRuntimeModal(NEW_JUPYTER_MODE)],
+      [NEW_RSTUDIO_MODE, () => renderNewRuntimeModal(NEW_RSTUDIO_MODE)],
+      [NEW_GALAXY_MODE, renderNewGalaxyModal],
+      [Utils.DEFAULT, renderCreateAnalysisBody]
+    )
+
+    const toolCardStyles = { backgroundColor: 'white', borderRadius: 5, padding: '1rem', display: 'inline-block', verticalAlign: 'middle', marginBottom: '1rem', textAlign: 'center', width: '100%', height: 60 }
+    const imageStyles = { verticalAlign: 'middle', height: '100%', width: '40%' }
+
+    const renderToolButtons = () => div({ style: { display: 'flex', alignItems: 'center', flexDirection: 'column', justifyContent: 'space-between' } }, [
+      div({ style: toolCardStyles, onClick: () => setViewMode(NEW_JUPYTER_MODE) }, [img({ src: jupyterLogoLong, style: _.merge(imageStyles, { width: '30%' }) })]),
+      div({ style: toolCardStyles, onClick: () => setViewMode(NEW_RSTUDIO_MODE) }, [img({ src: rstudioLogo, style: imageStyles })]),
+      div({ style: toolCardStyles, onClick: () => setViewMode(NEW_GALAXY_MODE) }, [img({ src: galaxyLogo, style: _.merge(imageStyles, { width: '30%' }) })])
+    ])
+
+    const renderCreateAnalysisBody = () => div({ style: { display: 'flex', flexDirection: 'column', flex: 1, padding: '.5rem 1.5rem 1.5rem 1.5rem' } }, [
+      renderToolButtons()
+    ])
+
+    const width = Utils.switchCase(viewMode,
+      [NEW_JUPYTER_MODE, () => 675],
+      [NEW_RSTUDIO_MODE, () => 675],
+      [NEW_GALAXY_MODE, () => 675],
+      [Utils.DEFAULT, () => 450]
+    )
+
+    const modalBody = h(Fragment, [h(TitleBar, {
+      title: 'Select an application',
+      titleStyles: _.merge(viewMode === undefined ? {} : { display: 'none' }, { marginLeft: '1.5rem', marginTop: '1.5rem' }),
+      width,
+      onDismiss,
+      onPrevious: !!viewMode ? () => setViewMode(undefined) : undefined
+    }),
+    hr({ style: { borderTop: '1px solid', width: '100%', color: colors.accent() } }),
+    getView(),
+    busy && spinnerOverlay]
+    )
```

Comment:
```suggestion
    const modalBody = h(Fragment, [
      h(TitleBar, {
        title: 'Select an application',
        titleStyles: _.merge(viewMode === undefined ? {} : { display: 'none' }, { marginLeft: '1.5rem', marginTop: '1.5rem' }),
        width,
        onDismiss,
        onPrevious: !!viewMode ? () => setViewMode(undefined) : undefined
      }),
      hr({ style: { borderTop: '1px solid', width: '100%', color: colors.accent() } }),
      getView(),
      busy && spinnerOverlay
    ])
```

Reply:
You may also want to `margin: '1.5rem 0 0 1.5rem'` if you don't mind specifying the other values.